src/server/pps/pretty/pretty.go

```diff
@@ -186,6 +189,78 @@ Job Counts:
 	return nil
 }
 
+// PrintDatumInfoHeader prints a file info header.
+func PrintDatumInfoHeader(w io.Writer) {
+	fmt.Fprint(w, "ID\tSTATUS\tTIME\t\n")
+}
```

Comment:
We could also present total in addition to those 3, and upload/download as well.

This seems like the question of where to present the data, a discussion which we have often in the API. In general, I think we choose to let the user specify what level of specificity they need by using the API. Those fields are printed in the `inspect-datum` view right now, so I think that's sufficient.